File: github_javapop/app/src/main/java/com/lfernando/githubjavapop/model/Repo.java
package com.lfernando.githubjavapop.model;

import com.google.gson.annotations.SerializedName;

public class Repo {
    @SerializedName("name")
    String name;

    @SerializedName("html_url")
    String url;

    @SerializedName("description")
    String description;

    @SerializedName("stargazers_count")
    int stars;

    @SerializedName("forks_count")
    int forks;

    @SerializedName("owner")
    User owner;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getStars() {
        return stars;
    }

    public void setStars(int stars) {
        this.stars = stars;
    }

    public int getForks() {
        return forks;
    }

    public void setForks(int forks) {
        this.forks = forks;
    }

    public User getOwner() {
        return owner;
    }

    public void setOwner(User owner) {
        this.owner = owner;
    }
}

File: app/src/main/java/com/desafioandroid/getirepos/data/dto/Repo.kt
package com.desafioandroid.getirepos.data.dto

import com.google.gson.annotations.SerializedName

data class Repo(
    @SerializedName("id")
    val id: Long,
    @SerializedName("node_id")
    val nodeId: String,
    @SerializedName("name")
    val name: String,
    @SerializedName("full_name")
    val fullName: String,
    @SerializedName("pulls_url")
    val pullsUrl: String,
    @SerializedName("stargazers_count")
    val stargazersCount: Long,
    @SerializedName("forks_count")
    val forksCount: Long,
    @SerializedName("description")
    val description: String?,
    @SerializedName("owner")
    val owner: Owner
)
